spec/controllers/socure_webhook_controller_spec.rb

```diff
@@ -63,6 +63,98 @@
         before do
           request.headers['Authorization'] = socure_secret_key
         end
+
+        shared_examples 'repeats webhooks' do
+          let(:headers) do
+            {
+              Authorization: socure_secret_key_queue.last,
+              'Content-Type': 'application/json',
+            }
+          end
+          let(:endpoints) do
+            (1..3).map { |i| "https://#{i}.example.test/endpoint" }
```

Comment:
WDYT of using this syntax instead? makes it easier to tell what the total count is, instead of having to worry about `..` vs `...` syntax
```suggestion
            3.times.map { |i| "https://#{i}.example.test/endpoint" }
```